config.yml

```diff
@@ -81,19 +81,38 @@ conditional-score-outputs:
     # Final score message supports using formatting strings for
     # scores.  Strings starting with "%" and matching a score name
     # will be interpolated: e.g. here the message will display the
-    # value of "score-1".
-    message: "1 wins! :)  Here is the score 1: %score-1"
+    # value of "score-standard".
+    message: "Perfect score! :)  You are absolutely ready to start the full assessment."
     condition-1: #free-form
-      # cas-1 is the case when score-& >= 1
-      score-1: 1
+      # cas-1 is the case when score-standards >= 10
+      score-standard: 10
   cas-2: # free-form
+    notification: "is-warning"
+    priority: 2
+    message: "Almost there! You have a few things to work on, but you should soon be ready to start an assessment."
```

Comment:
Let's make this more positive. "Almost there!" but "close enough to start an asssessment"